zerver/lib/bugdown/fenced_code.py

```diff
@@ -165,6 +165,11 @@ def generic_handler(processor: Any, output: MutableSequence[str],
     else:
         return CodeHandler(processor, output, fence, lang, run_content_validators)
 
+def remap_language(lang: str) -> str:
+    if lang in ['none', 'noop', 'text', 'plain']:
+        return ''
+    return lang
+
```

Comment:
Pygments already has a no-op lexer named `text`. We don’t need a custom check for this, and I think we should remove it for uniformity.